openff/benchmark/geometry_optimizations/compute.py

```diff
@@ -0,0 +1,55 @@
+"""
+Components for energy minimization / geometry optimization of molecules for FF benchmarking.
+
+These components leverage the QCArchive software stack for execution.
+
+In general, we abstract away much of the server's operational details, including datasets.
```

Comment:
From @jthorton : could choose a standard dataset name for benchmarking; could always add molecules to it. May be the better approach if the partners want to continue using the fractal server.